continuous_integration/docker/hadoop/files/etc/hadoop/conf.kerberos/mapred-site.xml

```diff
@@ -5,6 +5,7 @@
         <value>yarn</value>
     </property>
 
+    <!-- networking -->
```

Comment:
(I'm pretty sure we don't need this file at all)

Reply:
Figured as well, didn't dare go test if that would be ok or not but then I'll go for removal of it!